README.md

```diff
@@ -1,134 +1,104 @@
 # YouChoose.AI
 
-There are three components that compose this project.
+There are two components that compose this project.
 
 1. The browser extension. the core product of YouChoose.AI, sponsored by Ledger project round 3
 2. The backend of https://youtube.tracking.exposed, free software developed in 2018 under the ERC project ALEX (https://algorithms.exposed). This package since 2019 is maintained as an independent mechanism for Youtube algorithm accountability and enhanced to be the backend supporting YouChoose.
-3. The website https://youchoose.ai, is available at the address [TODO Ledgercommit]
 
 The backend in point 2 belongs to an existing repository that we link via git submodule.
 
-### Browser extension building
+## Browser extension building
 
-The extension building process would produce directory `build/` or `dist/`. The first contains the output for development and local execution, and the second creates the .zip for the public.
+The extension building process would produces a `build/` directory.
+It contains the browser extension for development as well as a the .zip file for the public.
 
-##### for developers and local testing.
+### For developers and local testing
 
 ```
 npm install
-npm run build
+npm run build:dev
+```
+
+Or, tu run the compilation in continuous mode while developing:
+
+```
+npm run watch
 ```
 
 it should produce an output like this:
 
 ```
-$ npm run build
-
-> youchoose.ai@0.1.2 build C:\Users\Claudio\VMShared\YCAI
-> rm -rf ./build && mkdir build && cross-env webpack && cp -r src/popup/* manifest.json icons/* ./build/
-
-NODE_ENV [undefined] Prod: false Devel:  true
-Development, using as environment variables: {"DEVELOPMENT":"true","NODE_ENV":"\"development\"","API_ROOT":"\"http://localhost:9000/api/v3\"","WEB_ROOT":"\"http://localhost:1313\"","VERSION":"\"0.1.2-dev\"","BUILD":"\"On the 08 of September pmt 14:36.\"","BUILDISODATE":"\"2021-09-08T12:36:50.782Z\"","FLUSH_INTERVAL":"10000"}
-Hash: cc4a9623ff47580332a5
-Version: webpack 2.1.0-beta.17
-Time: 8278ms
-        Asset     Size  Chunks             Chunk Names
-     popup.js  11.7 MB       0  [emitted]  popup
- dashboard.js  9.39 MB       1  [emitted]  dashboard
-       app.js     4 MB       2  [emitted]  app
-background.js  2.48 MB       3  [emitted]  background
-    + 825 hidden modules
+➡ npm run build:dev
```

Comment:
maybe it's not relevant to show the build "output"

Reply:
indeed